ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -36,18 +36,14 @@
 
   public static final String SINK_NUMBER_OF_PARTITIONS = "PARTITIONS";
   public static final String SINK_NUMBER_OF_PARTITIONS_PROPERTY = "ksql.sink.partitions";
-  public static final String DEFAULT_SINK_NUMBER_OF_PARTITIONS = "ksql.sink.partitions.default";
 
-  public static final String SINK_NUMBER_OF_REPLICATIONS = "REPLICATIONS";
-  public static final String SINK_NUMBER_OF_REPLICATIONS_PROPERTY = "ksql.sink.replications";
-  public static final String DEFAULT_SINK_NUMBER_OF_REPLICATIONS = "ksql.sink.replications.default";
+  public static final String SINK_NUMBER_OF_REPLICAS = "REPLICAS";
```

Comment:
These are not part of the config variables that can be set by the user via the KsqlConfig but they will be set in the query to specify the number of partitions, replications and the timestamp column for the result stream/table. This is just a constant to represent the name of the attributes. 
Here is an example:
`CREATE STREAM enrichedpv1 with (timestamp='pageid', partitions = 3) AS SELECT users.userid AS userid, pageid as pageid, region, gender FROM pageview LEFT JOIN users ON pageview.userid = users.userid;`

Reply:
Hmm. Then why is it in `KsqlConfig`? There is also a `SINK_NUMBER_OF_REPLICAS_PROPERTY` which I assume does go in the `ConfigDef`? It would be better to have the literals that are in the query moved to a separate file.

Reply:
Good idea, I'll move them to a new file, KsqlConstants.